src/functions/raisebox.js

```diff
@@ -28,16 +28,17 @@ defineFunction({
         };
     },
     htmlBuilder(group, options) {
-        const body = html.groupTypes.sizing({value: {
-            value: [{
-                type: "text",
-                value: {
-                    body: group.value.value,
-                    font: "mathrm", // simulate \textrm
-                },
-            }],
+        const text = new ParseNode("text", {
+            type: "text",
+            body: group.value.value,
+            font: "mathrm", // simulate \textrm
+        }, group.mode);
+        const sizedText = new ParseNode("sizing", {
+            type: "sizing",
+            value: [text],
             size: 6,                // simulate \normalsize
-        }}, options);
+        }, group.mode);
+        const body = sizing.htmlBuilder(sizedText, options);
```

Comment:
How about importing `sizingGroup` and
```js
const newOptions = options.havingSize(6); // simulate \normalsize
const body = sizingGroup([text], newOptions, options);
```
?

Reply:
That's true. The reason I did this was so that there's only one place the canonical implementation lives. That way, if it's ever updated in `sizing.js` we wouldn't have to remember to update it here as well. What do you think?

Reply:
@marcianx That seems right 😄